app/views/users/show.html.erb

```diff
@@ -15,4 +15,9 @@
 </div>
 <div class="col-md-4">
   <h4>Metrics</h4>
+    <ul>
+      <% for metric in @user.metrics %>
```

Comment:
@eliotsykes Okay, good. I had a feeling that was the case, but I couldn't actually remember what I had been doing in the past. Thanks!